docs/components/organization/organization-switcher.mdx

```diff
@@ -12,7 +12,9 @@ description: Clerk's <OrganizationSwitcher /> component is used to enable the ab
   src="/docs/images/ui-components/component-org_switcher.svg"
 />
 
-The [`<OrganizationSwitcher />`][orgswitcher-ref] component is used to enable the ability to switch between available organizations the user may be part of in your application.
+The [`<OrganizationSwitcher />`][orgswitcher-ref] component allows a user to switch between their accounts - their personal account and their organization account(s). This component is useful for applications that have a multi-tenant architecture, where users can be part of multiple organizations.
```

Comment:
> organization account(s)

Just want to make sure this is terminology we want - maybe something like `their personal account and their joined organizations` makes more sense?

Reply:
Sure! Let's update it

Reply:
```suggestion
The [`<OrganizationSwitcher />`][orgswitcher-ref] component allows a user to switch between their accounts - their personal account and their joined organizations. This component is useful for applications that have a multi-tenant architecture, where users can be part of multiple organizations.
```